change/@react-native-windows-telemetry-4abe7bcd-e858-4ab9-bc9c-3e2fd948acf2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
hmmm just curious, if you delete these change file and rerun "yarn change" do you get the same change file? I've never seen a type be "none" but that could do with it just changing the telemetry package!